### ISSUE TITLE
Unable to run with gradle 4.6

### ISSUE THREAD

**Issue by ketan** [jk1/Gradle-License-Report#77]
I see the following stacktrace in gradle 4.6 with the following configuration snippet.

The configuration is simply:

```groovy
  licenseReport {
    filters = new LicenseBundleNormalizer(bundlePath: rootProject.file('buildSrc/license-normalizer-bundle.json'))
  }
```

```plain
Caused by: groovy.lang.GroovyRuntimeException: Could not find matching constructor for: com.github.jk1.license.filter.LicenseBundleNormalizer$NormalizerLicenseBundle(com.github.jk1.license.filter.LicenseBundleNormalizer, groovy.json.internal.LazyValueMap)
        at com.github.jk1.license.filter.LicenseBundleNormalizer$_toConfig_closure15.doCall(LicenseBundleNormalizer.groovy:174)
        at com.github.jk1.license.filter.LicenseBundleNormalizer.toConfig(LicenseBundleNormalizer.groovy:174)
        at com.github.jk1.license.filter.LicenseBundleNormalizer.<init>(LicenseBundleNormalizer.groovy:52)
        at com.github.jk1.license.filter.LicenseBundleNormalizer.<init>(LicenseBundleNormalizer.groovy:41)
        at build_edej6x7wnrn759o1j0g7wr23f$_run_closure6$_closure23.doCall(/Users/ketanpadegaonkar/projects/gocd/gocd/build.gradle:283)
        at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:71)
        at org.gradle.util.ConfigureUtil.configureTarget(ConfigureUtil.java:160)
        at org.gradle.util.ConfigureUtil.configure(ConfigureUtil.java:106)
        at org.gradle.util.ConfigureUtil$1.execute(ConfigureUtil.java:123)
        at org.gradle.api.internal.plugins.ExtensionsStorage$ExtensionHolder.configure(ExtensionsStorage.java:183)
        at org.gradle.api.internal.plugins.ExtensionsStorage.configureExtension(ExtensionsStorage.java:67)
        at org.gradle.api.internal.plugins.DefaultConvention.configureExtension(DefaultConvention.java:399)
        at org.gradle.api.internal.plugins.DefaultConvention.access$500(DefaultConvention.java:45)
        at org.gradle.api.internal.plugins.DefaultConvention$ExtensionsDynamicObject.tryInvokeMethod(DefaultConvention.java:336)
        at org.gradle.internal.metaobject.CompositeDynamicObject.tryInvokeMethod(CompositeDynamicObject.java:98)
        at org.gradle.internal.metaobject.MixInClosurePropertiesAsMethodsDynamicObject.tryInvokeMethod(MixInClosurePropertiesAsMethodsDynamicObject.java:30)
        at org.gradle.internal.metaobject.ConfigureDelegate.invokeMethod(ConfigureDelegate.java:57)
        at build_edej6x7wnrn759o1j0g7wr23f$_run_closure6.doCall(/Users/ketanpadegaonkar/projects/gocd/gocd/build.gradle:282)
        at org.gradle.api.internal.ClosureBackedAction.execute(ClosureBackedAction.java:71)
```

**Comment by jk1** [comment]
@ketan what plugin version do you use?

**Comment by ketan** [comment]
Version 0.9.

**Comment by guenhter** [comment]
@ketan may I ask you for your `license-normalizer-bundle.json`?

**Comment by guenhter** [comment]
Add #78 to exclude some problems with inner classes (not sure if this fixes anything, but it should exclude some known groovy issues)

**Comment by jk1** [comment]
#78 has been merged and released in the scope of 1.0. Let's see if it helps.

**Comment by ketan** [comment]
Closing, the problem seems to have been fixed. Thanks for the quick fix.
